metadata.sh

```diff
@@ -68,7 +68,7 @@ export PROMETHEUS_EXPORTER_DOCKER_IMAGE="${PROMETHEUS_EXPORTER_DOCKER_IMAGE_NAME
 export INTEGRATION_TESTS_DOCKER_IMAGE_FROM="golang:1.13.1-stretch"
 export INTEGRATION_TESTS_DOCKER_IMAGE_NAMESPACE="mesosphere"
 export INTEGRATION_TESTS_DOCKER_IMAGE_NAME="kudo-cassandra-tests"
-export INTEGRATION_TESTS_DOCKER_IMAGE_TAG="latest"
+export INTEGRATION_TESTS_DOCKER_IMAGE_TAG="latest${IMAGE_DISAMBIGUATION_SUFFIX:-}"
```

Comment:
Nit: Let's remove `latest` here completely and use a different that than `latest` if `IMAGE_DISAMBIGUATION_SUFFIX` isn't set.

Reply:
TBH I'm not even sure why we're pushing this image at all :thinking: